app/src/main/res/layout/item_library.xml

```diff
@@ -1,18 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<androidx.cardview.widget.CardView
+<androidx.cardview.widget.CardView xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
   android:layout_margin="@dimen/card_margin"
   xmlns:android="http://schemas.android.com/apk/res/android">
 
   <androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/constraint_layout"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="?selectableItemBackground"
+    android:background="@color/card_view_background"
```

Comment:
does this still visually respond with a ripple on click?

Reply:
No @macgills the ripple does not seemed to work before only.

Reply:
because `?selectableItemBackground` is what contributes it. If you revert this and use the theme attribute suggested then it should all be fine

Reply:
No @macgills . Using the cardBackgroundColor will add the ripple effect.